### ISSUE TITLE
Add a --help option to get usage information

### ISSUE THREAD

**Issue by a2sheppy** [SphinxKnight/compat-tester#6]
Currently, specifying --help results in an error:

```
$ compat-tester --help
fs.js:651
  return binding.open(pathModule._makeLong(path), stringToFlags(flags), mode);
                 ^

Error: ENOENT: no such file or directory, open 'scope.json'
    at Object.fs.openSync (fs.js:651:18)
    at Object.fs.readFileSync (fs.js:553:33)
    at Object.<anonymous> (/Users/sheppy/.nvm/versions/node/v8.1.2/lib/node_modules/compat-tester/index.js:69:29)
    at Module._compile (module.js:569:30)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3)
    at Function.Module.runMain (module.js:605:10)
    at startup (bootstrap_node.js:158:16)
```

Ideally, this should provide usage information instead.

**Comment by SphinxKnight** [comment]
Thanks @a2sheppy :) Looking forward to solving those. Before v1 of course :)

**Comment by SphinxKnight** [comment]
Add https://www.npmjs.com/package/commander per the suggestion of @JeremiePat :)

**Comment by SphinxKnight** [comment]
Aaaand it's alive with 2ee9de29a9be1c9ed345abdc6e312eb52ed33b83 and published as 0.2.7 on NPM
